pallets/restricted-tokens/src/lib.rs

```diff
@@ -202,7 +202,7 @@ pub mod pallet {
 			+ LockableCurrency<Self::AccountId>
 			+ ReservableCurrency<Self::AccountId>
 			+ fungible::Inspect<Self::AccountId, Balance = Self::Balance>
-			+ fungible::InspectHold<Self::AccountId, Reason = ()>
+			+ fungible::InspectHold<Self::AccountId>
```

Comment:
Super :Warn::

We would need to migrate the storage from using `()` as an index of hold reason to any other variant. `()` is no longer supported 😢

Reply:
I will deal with this!

Reply:
Solved!